contrib/kind.sh

```diff
@@ -106,6 +132,17 @@ SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/12}
 NET_CIDR_IPV6=${NET_CIDR_IPV6:-fd00:10:244::/48}
 SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd00:10:96::/64}
 
+# Verify KIND_NUM_MASTER and KIND_NUM_WORKER match KIND_HA setting.
+if [ "$KIND_HA" == true ]; then
+  KIND_NUM_WORKER=0
```

Comment:
dont need to always set it to 0 here, just default it if it is not set via CLI.

Reply:
Because of the FOR loop in J2, needs to be set conditionally.

Reply:
Done